infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -708,9 +696,7 @@ public void unsubscribeReplicationEventTable(StreamListener listener) {
     }
 
     public void shutdown() {
-        if (runtimeTxLogging != null) {
-            runtimeTxLogging.shutdown();
-        }
```

Comment:
shutdown() handler's  a no-op? may I ask why?

Reply:
Because the runtime used here is shared so it is shutdown at the upper layer.. it was to used before to shutdown the specific Tx Log enabled runtime, but now that we've removed it, there is not any shutdown specific logic to trigger...